nexmo/__init__.py

```diff
@@ -174,10 +174,18 @@ def control_verification_request(self, params=None, **kwargs):
     return self.post(self.api_host, '/verify/control/json', params or kwargs)
 
   def get_basic_number_insight(self, params=None, **kwargs):
-    return self.get(self.api_host, '/number/format/json', params or kwargs)
+    return self.get(self.api_host, '/ni/basic/json', params or kwargs)
+
+  def get_standard_number_insight(self, params=None, **kwargs):
+    return self.get(self.api_host, '/ni/standard/json', params or kwargs)
 
   def get_number_insight(self, params=None, **kwargs):
-    return self.get(self.api_host, '/number/lookup/json', params or kwargs)
+    warnings.warn('nexmo.Client#get_number_insight is deprecated (use #get_standard_number_insight instead)', DeprecationWarning, stacklevel=2)
+
+    return self.get(self.api_host, '/ni/standard/json', params or kwargs)
```

Comment:
I think this method should still be using the deprecated endpoint (hence the warning in the same method)